migrations/11_set_project_image_name.js

```diff
@@ -0,0 +1,21 @@
+const Settings = require('settings-sharelatex')
+const mongojs = require('mongojs')
+const db = mongojs(Settings.mongo.url, ['users'])
```

Comment:
```suggestion
const db = mongojs(Settings.mongo.url, ['projects'])
```

Reply:
thanks

Reply:
I wonder what's the logic behind using a different collection. I copy-pasted it wrong from a different migration, but it worked just fine.

Reply:
https://github.com/mongo-js/mongojs/blob/df1fb993c379ff48537d3028f6722eaf6f5db014/index.js#L4-L27

It uses some type of lazy loading.